src/Configuration/Configuration.sln

```diff
@@ -1,111 +1,129 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
```

Comment:
Probably benign, but there's a lot more changes in this SLN than I'd expect (VS "clean-up"?).